internal/app/carrier/repository_test.go

```diff
@@ -0,0 +1,88 @@
+package carrier
+
+import (
+	"context"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"parcel-service/internal/app/model"
+	"testing"
+	"time"
+)
+
+func TestRepository_UpdateCarrierRequest(t *testing.T) {
+	var sourceTime = time.Now()
+	t.Run("should return success", func(t *testing.T) {
+		db, m, _ := sqlmock.New()
+		defer db.Close()
+
+		parcel := model.CarrierRequest{
```

Comment:
As Parcel struct is same for others, not need to assign same value  every time. You can assign this to beginning of the function.

Reply:
Thanks for the comment. Declare the parcel struct at the beginning of the function.